scripts/e2e-minikube.sh

```diff
@@ -9,6 +9,8 @@ setup_env() {
     echo "****** Installing and starting Minikube"
     scripts/installers/install-minikube.sh
 
+    readonly TEST_NAMESPACE="runtime-operator-test-${TEST_TAG}"
```

Comment:
It'll be good to use the short name rco-test- (to avoid exceeding character limits in some cases)